pygeoapi/util.py

```diff
@@ -30,29 +30,29 @@
 """Generic util functions used in the code"""
 
 import base64
-from typing import List
-from datetime import date, datetime, time
-from decimal import Decimal
-from enum import Enum
 import json
 import logging
 import mimetypes
 import os
-from pathlib import Path
 import re
+from datetime import date, datetime, time
+from decimal import Decimal
+from enum import Enum
+from pathlib import Path
 from typing import Any, IO, Union
-from urllib.request import urlopen
+from typing import List
 from urllib.parse import urlparse
-
-from shapely.geometry import Polygon
+from urllib.request import urlopen
 
 import dateutil.parser
-from jinja2 import Environment, FileSystemLoader, select_autoescape
-from babel.support import Translations
 import yaml
+from babel.support import Translations
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+from shapely.geometry import Polygon
```

Comment:
I merely re-ordered the imports of built-ins and installed libs here. Looked a bit chaotic...